ghost/core/test/integration/services/email-service/batch-sending.test.js

```diff
@@ -11,8 +11,10 @@ const escapeRegExp = require('lodash/escapeRegExp');
 const configUtils = require('../../../utils/configUtils');
 const {settingsCache} = require('../../../../core/server/services/settings-helpers');
 const DomainEvents = require('@tryghost/domain-events');
+const emailService = require('../../../../core/server/services/email-service');
 const should = require('should');
 const {mockSetting, stripeMocker} = require('../../../utils/e2e-framework-mock-manager');
+const Promise = require('bluebird');
```

Comment:
Can we please avoid using Bluebird? 🙏🏻 https://github.com/TryGhost/Ghost/issues/14882

Reply:
Yea no problem, I was going to tear it out anyways since it didn't seem to help :/